leaf-ui/js/onFunctions.js

```diff
@@ -632,104 +582,31 @@ function createHalo(cellView){
     halo.removeHandle('rotate');
 
 
-    halo.on('action:resize:pointermove', function(cell) {
+    halo.on('action:resize:pointermove', function() {
     	cellView.unhighlight();
 		cellView.highlight();
     });
 
+    // Remove halo when cell is unhighlighted
+    halo.options.cellView.on('removeHalo', function(){
+        halo.remove();
+    });
+
     halo.render();
     return halo;
```

Comment:
Also will remove this return as it is no longer needed (comments for myself as well)